README.md

```diff
@@ -38,64 +69,66 @@ Add the following importmap to your page anywhere before you use the library:
 {
   "imports": {
     "react": "https://esm.sh/react@18",
-    "react-dom": "https://esm.sh/react-dom@18"
+    "react-dom": "https://esm.sh/react-dom@18",
+    "react-dom/client": "https://esm.sh/react-dom@18/client"
   }
 }
 </script>
 ```
 
-### 4. Import and use the library and any other functions you need in an ES module script:
+### 4. Import the library and instantiate the component with React in an ES module script:
 
 ```html
 <script type="module">
-  import { BlueskyComments } from 'https://unpkg.com/bluesky-comments@0.4.0/dist/bluesky-comments.es.js';
-  document.addEventListener('DOMContentLoaded', function() {
-    const author = 'you.bsky.social';
-    if (author) {
-      BlueskyComments.init('bluesky-comments', {author});
+  import { createElement } from 'react';
+  import { createRoot } from 'react-dom/client';
+  import { BlueskyComments } from 'https://unpkg.com/bluesky-comments@<VERSION>/dist/bluesky-comments.es.js';
+
+  const author = 'you.bsky.social';
+  const container = document.getElementById('bluesky-comments');
+  const root = createRoot(container);
+  root.render(
+    createElement(BlueskyComments, {
+      author: author,
     }
-  });
+  );
 </script>
 ```
 
-See the [Usage](#usage) section for details on the API.
+See the [Usage](#usage) section below for details on the API.
 
-### (Deprecated) Installation using `<script>` tags and UMD
+## Usage
 
-Previous versions of this library recommended installing like this:
+Examples in this section use the React JSX syntax. If you're using a non-React project, you can
+instead use the `createElement` function and pass the react options in.
 
-```html
-<script src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
-<script src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"></script>
-<script src="https://unpkg.com/bluesky-comments@<VERSION>/dist/bluesky-comments.umd.js"></script>
-```
+For example, the following two examples are equivalent:
 
-And initializing the comments in a standard `<script>` tag. Both of these approaches work:
+React JSX:
 
-```html
-<script>
-  document.addEventListener('DOMContentLoaded', function() {
-    const uri = 'https://bsky.social/coryzue.com/posts/3jxgux';
-    if (uri) {
-      // New API
-      BlueskyComments.init('bluesky-comments', {uri});
-
-      // Legacy API (still supported but deprecated)
-      initBlueskyComments('bluesky-comments', {uri});
-    }
-  });
-</script>
+```javascript
+<BlueskyComments
+  author="you.bsky.social"
+  uri="https://bsky.app/profile/coryzue.com/post/3lbrko5zsgk24"
+/>
 ```
 
-This option is now deprecated with the introduction of ES modules and will be removed in a future version.
+Non-React:
```

Comment:
Again, this is all "react" based. I'd say "Non JSX" (as an h3?)
```
### Non JSX
```

And above "React JSX" also as an h3?

Reply:
thanks 8487bbf4f52c154aebbdc6cc4e617d646403d01b